prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -64,8 +64,8 @@ public async Task SendProvisionerLinkAsync(string[] emails, EnrolmentCertificate
                 .Where(e => e.Id == token.EnrolleeId)
                 .Select(e => new
                 {
-                    e.FirstName,
-                    e.LastName,
+                    FirstName = e.PreferredFirstName ?? e.FirstName,
+                    LastName = e.PreferredLastName ?? e.LastName,
```

Comment:
Should this be refactored into common functions or property?

Reply:
Sure, but don't know sure what the property name should be. Any idea?

Reply:
What do you think of `Current`?

Reply:
The code is updated.